src/components/Table/views/Tree/index.tsx

```diff
@@ -351,45 +364,36 @@ class TreeView extends Component<Props, State> {
                   },
                 }}
               >
-                {currentNode?.id}: {currentNode?.name}
+                {currentNode?.name}
               </Link>
             </header>
             {currentNode?.rank?.toLowerCase() !== 'no rank' && (
               <div>
-                <Tooltip title="Rank.">
-                  <span
-                    className={css('small', 'icon', 'icon-common')}
-                    data-icon="&#xf129;"
-                    aria-label="Rank."
-                  />
-                </Tooltip>{' '}
-                <i>{currentNode?.rank}</i>
+                <p>{currentNode?.rank}</p>
               </div>
             )}
             {currentNode?.lineage && (
```

Comment:
Only show breadcrumbs when we have more than one item:

```suggestion
            {currentNode?.lineage && lineageIDs.length > 1 && (
```